newm-server/src/main/kotlin/io/newm/server/features/song/repo/SongRepositoryImpl.kt

```diff
@@ -46,9 +57,6 @@ internal class SongRepositoryImpl(
                 streamUrl = song.streamUrl
                 nftPolicyId = song.nftPolicyId
                 nftName = song.nftName
-                mintingStatus = song.mintingStatus ?: mintingStatus
-                marketplaceStatus = song.marketplaceStatus ?: marketplaceStatus
-                this.paymentKeyId = song.paymentKeyId?.let { EntityID(it, KeyTable) } ?: paymentKeyId
```

Comment:
Don't want to allow the setting these values when the User create the Song through the REST API.